share/completions/cargo.fish

```diff
@@ -3,7 +3,7 @@
 ## --- WRITTEN MANUALLY ---
 set -l __fish_cargo_subcommands (cargo --list 2>&1 | string replace -rf '^\s+([^\s]+)\s*(.*)' '$1\t$2' | string escape)
 
-# Append user-installed extensions (e.g. cargo-foo, invokable as `cargo foo`) to the list of subcommands (à la git)
+# Append user-installed extensions (e.g. cargo-foo, invocable as `cargo foo`) to the list of subcommands (à la git)
```

Comment:
I reckon either is acceptable?

Reply:
You are right. Both are valid, so my suggestion too 😅

Also there is a discussion about this exact debate in the tool I used to detect/fix typo

- https://github.com/crate-ci/typos/issues/1065

So I would say, it's up to you.

I can either change it, or we leave it like that, and the PR can be merged as is, considering my [reply](https://github.com/fish-shell/fish-shell/pull/11069#discussion_r1935408675) on "busses"

Reply:
let's run with "invocable"